dev-python/tlsh/files/tlsh-4.8.2-setup-sources.patch

```diff
@@ -0,0 +1,30 @@
+--- a/setup.py
++++ b/setup.py
+@@ -21,26 +21,7 @@ with open(join(realpath('..'), 'CMakeLists.txt'), 'r') as f:
+     if m:
+       tlsh_3b = ''
+ 
+-if os.name == 'nt':
+-  tlsh_module = Extension('tlsh', \
+-    sources = ['tlshmodule.cpp', \
+-      join(realpath('..'), 'src', 'tlsh.cpp'), \
+-      join(realpath('..'), 'src', 'tlsh_impl.cpp'), \
+-      join(realpath('..'), 'src', 'tlsh_util.cpp') \
+-    ], \
+-    include_dirs = [join(realpath('..'), 'include'),
+-                    join(realpath('..'), 'Windows')],\
+-    define_macros = [('WINDOWS', None),], \
+-  )
+-else:
+-  tlsh_module = Extension('tlsh', \
+-    sources = ['tlshmodule.cpp', \
+-      join(realpath('..'), 'src', 'tlsh.cpp'), \
+-      join(realpath('..'), 'src', 'tlsh_impl.cpp'), \
+-      join(realpath('..'), 'src', 'tlsh_util.cpp') \
+-    ], \
+-    include_dirs = [join(realpath('..'), 'include')],
+-  )
++tlsh_module = Extension('tlsh', sources=['tlshmodule.cpp'], libraries=['tlsh'])
```

Comment:
I wonder what the sources= thing is for, if we're using the system copy. Or do I misunderstand the point of the patch?

(Is it just to fix the layout?)

Reply:
My understanding is that this is code that creates the python module that can call the shared lib. The module is originally built to include the entire tlsh code in it, but the tlshmodule.cpp only sets up the bindings from python to c++.

Reply:
Oh okay.